curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98de.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-22
 
 # --description--
 
-Create a new variable below the other one called `--building-color2` and give it a value of `#66cc99`. Then set it as the `background-color` of `.bb2`.
+Create a new variable below your `--building-color1` variable. Name your new variable `--building-color2` and give it a value of `#66cc99`.
```

Comment:
```suggestion
Create a new variable below your `--building-color1` variable. Name your new variable `--building-color2` and give it a value of `#66cc99`. Then set it as the `background-color` of `.bb2`.
```
Shouldn't the last sentence be kept? Otherwise description doesn't mention using created variable, what is checked by tests.

Reply:
Whoops! Good catch.